app/uk/gov/hmrc/disareturns/models/errors/connector/responses/UpstreamErrorMapper.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.disareturns.models.errors.connector.responses
+
+import play.api.Logging
+import uk.gov.hmrc.http.UpstreamErrorResponse
+
+object UpstreamErrorMapper extends Logging {
```

Comment:
Test spec?

Reply:
Missed this apologies, we've added it in now.